docs/release-notes/release_02-2023.md

```diff
@@ -1,52 +1,61 @@
-# Welcome to the February 2023 OpenScienceLab Update!
+# February 2023
+
+Welcome to the February 2023 OpenScienceLab Update!
 
 ### Changes:
 - [OpenScienceLab](#opensciencelab)
 - [Single Sign On](#single-sign-on)
 - [Automatic Authentication](#automatic-authentication)
 - [User Access](#user-access)
 - [IP Filter](#ip-filter)
-- [User Request]()
+- [User Request](#user-resquest)
 - [Multi Download](#multi-download)
 - [Kernel Usage](#kernel-usage)
 - [Recommended Jupyter Notebook Changes](#recommended-jupyter-notebook-changes)
 
 ---
-  
+
+(opensciencelab)=
```

Comment:
This must be the `myst` markdown syntax that doesn't render right? Does `myst` not support the normal anchor referencing? Obviously from a normal MD standpoint, this is not necessary and is in fact making rendering worse.

Reply:
I was going to resolve this after reading about named anchors in Myst. BUT Myst should also support regular MD anchors. Perhaps these are not actually necessary?

Reply:
The reason for keeping the anchors here is related to how internal links and link preview modals interact with header text. 

If your header includes any special formatting like code blocks, italics, or bold text, links and preview modals won't work. Using anchors allows you to format the headers however you like. 

These anchors are also used to link sections inside entire Myst MD projects. If you were rendering a Jupyter Book to serve as a static website, you could have preview modals on links to things in other `md` docs. We probably won't use that feature because it won't work in an unrendered Juypter Book being viewed in JupyterLab. Just mentioning it as an FYI.